packages/nodejs/src/config/configmap.ts

```diff
@@ -80,6 +84,10 @@ export const JS_TO_RUBY_MAPPING: { [key: string]: string } = {
   ignoreActions: "ignore_actions",
   ignoreErrors: "ignore_errors",
   ignoreNamespaces: "ignore_namespaces",
+  instrumentHttp: "instrument_http",
+  instrumentHttps: "instrument_https",
```

Comment:
```suggestion
```

Since the HTTP and HTTPS settings have been merged into `instrumentHttp`, the `instrumentHttps` entry can be removed from this map.